src/lib.rs

```diff
@@ -196,11 +195,37 @@ pub fn initialize(_argc: *const isize, _argv: *const *const *const u8) -> isize
 ///
 /// You can also enable the `arbitrary` crate's custom derive via this crate's
 /// `"arbitrary-derive"` cargo feature.
+/// 
+/// ## Init Code
+/// 
+/// Init code to the fuzz target by using the `init` keyword. This is called once before the fuzzer starts.
+/// Supports short |input| or |input: <type>| syntax.
+/// 
+/// ```no_run
+/// #![no_main]
+/// 
+/// use libfuzzer_sys::fuzz_target;
+/// 
+/// fuzz_target!(init: (), |input| {
+///     // ...
+/// });
```

Comment:
If we're going to have an example of using `init:`, can we make it a little more realistic / helpful for readers?

```suggestion
/// #![no_main]
/// 
/// use libfuzzer_sys::fuzz_target;
/// use std::collections::HashSet;
/// use std::sync::OnceLock;
///
/// static DICTIONARY: OnceLock<HashSet<String>> = OnceLock::new();
///
/// fuzz_target!(
///     init: {
/// #       leat read_dictionary = |_| unimplemented!();
///         let dictionary = read_dictionary("/usr/share/dict/words");
///         DICTIONARY.set(dictionary).unwrap();
///     },
///     |input| {
///         // Use the initialized `DICTIONARY` here...
///     }
/// );
```